modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/output/TableFormatter.java

```diff
@@ -48,11 +48,12 @@ public void printExchanges(Exchange[] exchanges) {
                                           .max()
                                           .getAsInt();
 
-        int maxColumnSize = Math.max(maxExchangeNameLength, Exchange.NAME.length());
+        int maxColumnSize = Math.max(maxExchangeNameLength, Exchange.NAME_TAG.length());
 
-        String printTemplate = "%-" + String.valueOf(maxColumnSize + TABLE_PADDING) + "s%-10s%-10s%-10s\n";
+        String printTemplate = "%-" + maxColumnSize + TABLE_PADDING + "s%-10s%-10s%-10s\n";
```

Comment:
Maybe we need to use parentheses here to make it more readable since we have removed the `String.valueOf`.

Reply:
Fixed.